templates/pwm.asg.element.json

```diff
@@ -0,0 +1,843 @@
+								"content": {
+									"Fn::Join": [
+										"",
+										[
+											"base:\n",
+											"  '*':\n",
+											"    - ",
+											{
+												"Fn::Select":[  
+													"0",
+													{
+														"Fn::Split" : [ 
+															".", 
+															{ 
+																"Ref": "SaltPillarFile" 
+															} 
+														]
+													}
+												]
+											},
+											"\n",
+											"    - extras"
+										]
+									]
+								},
+								"group": "root",
+								"mode": "000700",
+								"owner": "root"
+							},
+							"/srv/salt/pillar/extras.sls": {
+								"content": {
+									"Fn::Join": [
+										"",
+										[
+											"pwm:\n",
+											"  lookup:\n",
+											"    config_bucket: ",
+											{
+												"Ref": "ConfigBucketName"
+											}
+										]
+									]
+								},
+								"group": "root",
+								"mode": "000700",
+								"owner": "root"
+							},
+							"/etc/salt/minion.d/pwm.conf": {
+								"content": {
+									"Fn::Join": [
+										"",
+										[
+											"file_client: local\n",
+											"file_roots: \n",
+											"  base:\n",
+											"  - /srv/salt/\n",
+											"pillar_roots: \n",
+											"  base:\n",
+											"  - /srv/salt/pillar/\n",
+											"hash_type: sha512\n",
+											"pillar_merge_lists: true\n",
+											"log_file: /var/log/salt/minion\n"
+										]
+									]
+								},
+								"group": "root",
+								"mode": "000700",
+								"owner": "root"
+							}
+						},
+						"packages": {
+							"yum": {
+								"git": []
+							}
+						}
+					},
+					"c-adminusersetup": {
+						"commands": {
+							"10-cp_adminsshusermgmt": {
+								"command": "cp /srv/salt/pwm/adminusermgmt.sh /usr/local/bin/adminusermgmt.sh"
+							},
+							"20-ch_adminsshusermgmt": {
+								"command": "chmod 700 /usr/local/bin/adminusermgmt.sh & chown root /usr/local/bin/adminusermgmt.sh & chgrp root /usr/local/bin/adminusermgmt.sh"
+							},
+							"30-configure_sshd_command": {
+								"command": "sed -i 's:#AuthorizedKeysCommand none:AuthorizedKeysCommand /usr/local/bin/authorized_keys_command.sh:g' /etc/ssh/sshd_config"
+							},
+							"40-configure_sshd_commanduser": {
+								"command": "sed -i 's:#AuthorizedKeysCommandUser nobody:AuthorizedKeysCommandUser nobody:g' /etc/ssh/sshd_config"
+							},
+							"50-runadminsshusermgmt": {
+								"command": "/usr/local/bin/adminusermgmt.sh -G $(cat /usr/local/bin/admingroupname)"
+							}
+						},
+						"files": {
+							"/etc/cron.d/adminusermgmt": {
+								"content": {
+									"Fn::Join": [
+										"",
+										[
+											"*/8 * * * * root /usr/local/bin/adminusermgmt.sh -G $(cat /usr/local/bin/admingroupname)",
+											"\n"
+										]
+									]
+								},
+								"group": "root",
+								"mode": "000644",
+								"owner": "root"
+							},
+							"/usr/local/bin/admingroupname": {
```

Comment:
Not necessary if a ref is used where this file is currently cat'd...

Reply:
The reason I was leaving it there was for the extra scripts that were originally included in the formula. Now that those extra scripts are being removed from the box, I don't think this will be needed anymore